src/goGetImport.ts

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+import vscode = require('vscode');
+import cp = require('child_process');
+import path = require('path');
+import fs = require('fs');
+import { getGoRuntimePath } from './goPath';
+import { getCurrentGoPath, getImportPath } from './util';
+
+export function goGetImport() {
+	const editor = vscode.window.activeTextEditor;
+	const selection = editor.selection;
+	const selectedText = editor.document.lineAt(selection.active.line).text;
+
+	const importPath = getImportPath(selectedText);
+
+	const goRuntimePath = getGoRuntimePath();
+	if (!goRuntimePath) {
+		return vscode.window.showErrorMessage('Could not locate Go binaries. Make sure you have Go installed');
+	}
+
+	fs.exists(path.join(getCurrentGoPath(), 'src', importPath), (exists) => {
```

Comment:
we can skip this check and run `go get -u` regardless
If the package already exists then it will be updated.
Thoughts?

Reply:
I think it's a good idea, if we make sure to let the user know that their package will be updated. They might not want that for whatever reason.